CONTRIBUTING.md

```diff
@@ -11,6 +11,38 @@ and smoother if you have the time and energy to read and follow these suggestion
 
 There's never enough!  Pretty much anything is fair game to improve here.
 
+### Running the doc build yourself
+
+To check your documentation additions, you can optionally build the docs yourself and see how the changes will look on the webpage. 
+
+To run the doc build yourself, the setup step is:
+```
+$ cargo install --vers "^0.4" mdbook
+$ cargo install mdbook-linkcheck
+$ cargo install mdbook-mermaid
+$ cargo install gelatyx --version "^0.2"
```

Comment:
I don't believe that this is accurate any more; we don't use mdbook any longer so those first 3 are no longer needed.  `build-docs.sh` itself should install gelatyx if needed, so I think this section can probably be removed.

Reply:
Okay, I will remove this then. It was based on a comment you wrote in another pull request in January.